.gitignore

```diff
@@ -1,463 +1,464 @@
-/IDE/ARDUINO/library.properties.tmp.backup
-/IDE/ARDUINO/PREPENDED_README.md
-
-# Doxygen generated files
-doc/doxygen_warnings
-doc/html
-doc/pdf
-
-# XCODE Index
-IDE/XCODE/Index
-IDE/**/xcshareddata
-IDE/**/DerivedData
-
-# ARM DS-5 && Eclipse
-\.settings/
-\.cproject
-\.project
-\.autotools
-
-# Renesas e2studio
-/IDE/Renesas/e2studio/Projects/test/src/smc_gen
-/IDE/Renesas/e2studio/Projects/test/trash
-/IDE/Renesas/e2studio/Projects/test/*.launch
-/IDE/Renesas/e2studio/Projects/test/*.scfg
-
-/IDE/Renesas/e2studio/RX65N/GR-ROSE/.metadata
-/IDE/Renesas/e2studio/RX65N/GR-ROSE/smc/src
-/IDE/Renesas/e2studio/RX65N/GR-ROSE/smc/trash
-/IDE/Renesas/e2studio/RX65N/GR-ROSE/test/src/smc_gen
-/IDE/Renesas/e2studio/RX65N/GR-ROSE/test/generate
-
-/IDE/Renesas/e2studio/RX65N/RSK/.metadata
-/IDE/Renesas/e2studio/RX65N/RSK/smc/src
-/IDE/Renesas/e2studio/RX65N/RSK/smc/trash
-/IDE/Renesas/e2studio/RX65N/RSK/test/src/smc_gen
-/IDE/Renesas/e2studio/RX65N/RSK/test/generate
-
-/IDE/Renesas/e2studio/RX72N/EnvisionKit/.metadata
-/IDE/Renesas/e2studio/RX72N/EnvisionKit/smc/src
-/IDE/Renesas/e2studio/RX72N/EnvisionKit/smc/trash
-/IDE/Renesas/e2studio/RX72N/EnvisionKit/test/src/smc_gen
-/IDE/Renesas/e2studio/RX72N/EnvisionKit/test/generate
-
-# QNX CAAM
-/IDE/QNX/example-server/server-tls
-/IDE/QNX/example-client/client-tls
-/IDE/QNX/example-cmac/cmac-test
-/IDE/QNX/CAAM-DRIVER/wolfCrypt
-
-# Xilinx
-/IDE/XilinxSDK/data
-
-# Emacs
-*~
-
-# CMake
-CMakeFiles/
-CMakeCache.txt
-cmake_install.cmake
-
-# GDB Settings
-\.gdbinit
-
-libFuzzer
-
-# Pycharm and other IDEs
-\.idea
-
-# FIPS
-XXX-fips-test
-
-# ASYNC
-/wolfAsyncCrypt
-/async
-
-# Generated user_settings_asm.h.
-user_settings_asm.h
-
-# VisualGDB
-**/.visualgdb
-
-# Espressif sdk config default should be saved in sdkconfig.defaults
-# we won't track the actual working sdkconfig files
-/IDE/Espressif/**/sdkconfig
-/IDE/Espressif/**/sdkconfig.old
-
-# ESP8266 RTOS SDK has a slightly different sdkconfig filename to exclude:
-/IDE/Espressif/**/sdkconfig.debug
-/IDE/Espressif/**/sdkconfig.release
-
-# Always include Espressif makefiles (typically only used for ESP8266)
-!/IDE/Espressif/**/Makefile
-!/IDE/Espressif/**/component.mk
-
-# MPLAB
-/IDE/MPLABX16/wolfssl.X/dist/default/
-/IDE/MPLABX16/wolfssl.X/.generated_files
-/IDE/MPLABX16/wolfcrypt_test.X/dist/default/
-/IDE/MPLABX16/wolfcrypt_test.X/.generated_files
-
-# auto-created CMake backups
-**/CMakeLists.txt.old
-
-# MagicCrypto (ARIA Cipher)
-MagicCrypto
-
-# CMake  build directory
-/out
-/out_temp
-
-# debian packaging
-debian/changelog
-debian/control
-*.deb
-
-# Ada/Alire files
-wrapper/Ada/alire/
-wrapper/Ada/config/
-wrapper/Ada/lib/
-wrapper/Ada/obj/
-
-# PlatformIO
-/**/.pio
-/**/.vscode/.browse.c_cpp.db*
-/**/.vscode/c_cpp_properties.json
-/**/.vscode/launch.json
-/**/.vscode/ipch
-/**/sdkconfig.esp32dev
-
-# Autogenerated debug trace headers
-wolfssl/debug-trace-error-codes.h
-wolfssl/debug-untrace-error-codes.h
+ctaocrypt/src/src/

```

Comment:
Seems like you may have introduced CRLF into this file. Please resolve.

Reply:
During PR creation, there was no preview warning that there would be a merge conflict. There were petty changes needed so I tried to resolve them with the offered "edit in web" feature. That seems to have pulled in ALL of the changes from master in a new commit, which was specifically what I did not want.

I'm not sure how to proceed without adding a new commit to fix the CRLF from Visual Studio. I could close this one and start over? If I squash, all the specific changes will be lost and new changes included. Only my branch would then reflect the actual published code.

Reply:
> Seems like you may have introduced CRLF into this file. Please resolve.

ok, @dgarske I'm getting better at recovering from git tangle. 

I've fixed up this PR to exclude the problematic `.gitignore`, and two other petty files with only spell checks that were causing a merge conflict.

See additional PR comments, below.